contrib/ohi-release-notes/run.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+set -euo pipefail
+
+RT_PKG="github.com/newrelic/release-toolkit@latest"
```

Comment:
No. The action and the script are two different things.

The action runs the `action.yml` that runs other actions so it stays with the same version that the action has as it is already downloaded. Here you have the line I am referring to: https://github.com/newrelic/release-toolkit/pull/135/files#diff-3435321f89580c19c1ccf925d4992905d93df0e2f863bff8b1ad6261e0cdb7d2R29-R35

note that the `uses` is local because it is already downloaded.

The script is meant to be used locally on our laptops for manual usage. I have to kid you and say RTFM: https://github.com/newrelic/release-toolkit/pull/135/files#diff-286a0a98212926ebee9d0bcce9e6bba72c6987271fbde8b5a5a94c2d9098ad2fR34-R40

Reply:
Oh, I reviewed it quickly and I missed that, then why having twice the same logic? Can't the action call the script?

I am a bit scared by possible discrepancies: i.e. "There is a run.sh script that **should do** the same as this action"

However, if you discussed it already multiple times and that was the best option I am fine with it, I just find it a bit weird and a possible source of bugs

Reply:
It is inspired by the infrastructure-bundle: https://github.com/newrelic/infrastructure-bundle/blob/4864fbd/run-ci-locally.sh

GitHub gives us a CI but not a way to run it locally. The project `act` (https://github.com/nektos/act) has a good project and it is growing really fast but it is still not a viable solution. So we have a YAML to run an action and another script to run it locally.

There is nothing more we can do. The script also works different from the action as the action needs outputs while the script simply leaves the files to be used and prints the title so the user can _do things_ with that.

Reply:
yes, I agree that running the action locally is not desirable. 

What I meant was having the action running the same script that we can use locally, something like:

```
inputs:
[...]
runs:
  using: "composite"
  steps:
    - run: ./run.sh
      shell: bash
```

So that we are sure that the logic is the same. The inputs could still not be mapped correctly, but at least everything else would be the same and we could remove a whole file. 

However, as I said before I do not want to be a blocker for this in case you have discussed for a while the whole behavior  😄

Reply:
Solved with an offline conversation.

> There are so many differences between [running outside and inside the CI] that it is better to have directly two codebases.